carrot_cli/src/carrot_cli/template/command.py

```diff
@@ -129,37 +129,46 @@ def find(
     "--pipeline",
     "--pipeline_id",
     help="The ID or name of the pipeline that will be this template's parent",
-    required=True,
+    default=""
```

Comment:
For these options you can add `type=str` and `default=None`, then you can check for whether they're `None` below rather than using the empty string as the sentinel.

Reply:
That makes sense.  Just pushed a commit with that change